app/routes/__index/domains/index.tsx

```diff
@@ -51,6 +51,12 @@ export const action = async ({ request }: ActionArgs) => {
     });
   }
 
+  if (record.status !== 'active') {
+    throw new Response('Record is not active, action forbidden', {
+      status: 403,
```

Comment:
I wonder about using a `403` here.  A `403` implies authorization failure vs. an invalid resource state.  I think this is a `409`: https://http.dev/409

Reply:
I used 403 because the chatGPT suggested so 😭 

<img width="949" alt="image" src="https://user-images.githubusercontent.com/32075241/226222313-9e01ac01-6fed-4f77-9c86-8b8d36832cbc.png">